plugins/scaffolder-backend/src/service/router.ts

```diff
@@ -331,7 +335,21 @@ export async function createRouter(
       });
 
   actionsToRegister.forEach(action => actionRegistry.register(action));
-  workers.forEach(worker => worker.start());
+
+  if (options.eventBroker) {
```

Comment:
let's drop this for now, and if there's no lifecycle service just start the workers.